Gonryu patch write score (#17)
icon-project/documentation: score/writing-score.md

## Purpose 

You can understand how to write SCORE.

## Prerequisite 

* [score-overview](https://github.com/icon-project/documentation/blob/master/score/score-overview.md)
* [score-by-example](https://github.com/icon-project/documentation/blob/master/score/score-by-example.md)
* [t-bears-tutorial](https://github.com/icon-project/t-bears)

## Creating Workspace

SCORE is written in Python programming language, so you can use any tools for programming for Python, as long as the required files, as described below,  are included in the SCORE package. These files are all text.

> * \_\_init\_\_.py: source codes written in Python to initialize a SCORE.

Reviewer: ```suggestion
* \_\_init\_\_.py: Indicating the directory as containing packages.
```

> * project.py: source codes of a SCORE, written in Python. The file name is written in package.json.

Reviewer: ```suggestion
* _project.py_: The main module file of the SCORE to be executed at the top level and this should be specified in `package.json`.
```

> * package.json: configurations of a SCORE.

Reviewer: ```suggestion
* package.json: Providing basic information of the SCORE.
```
